app/parsers/bulkrax/csv_spot_parser.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'csv'
+module Bulkrax
```

Comment:
could you rename this to `Spot::CsvParser` (and rename the folder to match)? since we're subclassing the parser as our own, I think I'd like to keep it in the Spot namespace. (you'll need to update the Bulkrax initializer)

so:

```ruby
module Spot
  class CsvParser < Bulkrax::CsvParser
  # ....
  end
end